subcommand_plugin_examples/c_subcommand_plugin_tutorial/README.md

```diff
@@ -0,0 +1,243 @@
+        name="temp-converter",
+        summary="A custom subcommand written in C that converts Celsius to Fahrenheit",
+        action=conda_temp_converter,
+    )
+```
+
+## Packaging the custom subcommand
+
+In order to install the `conda temp-converter` custom subcommand we will need to configure a Python build system. You can either use the [PEP 621][pep 621] compliant `pyproject.toml` or alternatively `setup.py` can be used (not shown in this tutorial):
+
+```toml
+[build-system]
+requires = ["setuptools>=61.0", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "temp-converter"
+version = "1.0"
+description = "A custom subcommand written in C that converts Celsius to Fahrenheit"
+requires-python = ">=3.7"
+dependencies = ["conda"]
+
+[tools.setuptools]
+py_modules=["temp-converter"]
+
+[project.entry-points.conda]
+temp-converter = "temp_conv_c"
+```
+
+<details>
+<summary><code>pyproject.toml details</code></summary>
+
+> #### `[build-system]`
+> - `requires` This is a list of requirement specifiers for build-time dependencies of a package.
+> - `build-backend` Build backends have the ability to accept configuration settings, which can change the way that the package building is handled.
+> 
+> #### `[project]`
+> * `name` (required) This is the name of the package that contains your subcommand. This is also how others will find your subcommand package if you choose to upload it to PyPI.
+> * `version` (required) The version of the project; can be specified *either* statically or listed as dynamic.
+> `description` A brief description of the project.
+> * `requires-python` The version(s) of Python required by your project.
+> * `dependencies` These are all of the dependencies for your project. This specific subcommand example requires `conda`, which is why it is listed here.
+>
+> For more information on `pyproject.toml` see the [PyPA packaging documentation][pyproject.toml docs].
+
+</details>
+
+
+> **Note:**
+> For more information about entry points specification in general, please read [PyPA's entrypoints documentation][entrypoints docs].
+
+### Development/Editable Install
+
+The custom `temp-converter` subcommand plugin can be installed as an editable install by running the following command:
+
+
+```bash
+pip install -e .
+```
+
+### Conda Install
+
+When you're ready to distribute your custom `temp-converter` subcommand plugin you can package it as a conda package:
+
+<details>
+<summary><code>recipe/meta.yaml</code></summary>
+
+```yaml
+package:
+  name: temp-converter
+  version: 1.0
+
+source:
+  path: ../
+
+build:
+  script: $PYTHON -m pip install --no-deps .
+
+requirements:
+  host:
+    - python >=3.7
+
+  run:
+    - conda
+    - python >=3.7
+
+about:
+  home: https://github.com/conda/conda-plugin-template
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: A custom subcommand written in C that converts Celsius to Fahrenheit
+```
+
+</details>
+
+```bash
+$ conda build ./recipe
```

Comment:
I think it would be a good idea to link to how to upload these to anaconda.org. Some of the plugin authors may not know that this could be a good way to distribute their plugins, or even know what anaconda.org is.